src/preprocess.py

```diff
@@ -228,35 +228,37 @@ def build_tasks(args):
     '''
 
     # 1) create / load tasks
-    tasks, train_task_names, eval_task_names = \
-        get_tasks(parse_task_list_arg(args.pretrain_tasks),
-                  parse_task_list_arg(args.target_tasks), args.max_seq_len,
-                  path=args.data_dir, scratch_path=args.exp_dir,
-                  load_pkl=bool(not args.reload_tasks),
-                  nli_prob_probe_path=args['nli-prob'].probe_path,
-                  max_targ_v_size=args.max_targ_word_v_size)
+    tasks, train_task_names, eval_task_names = get_tasks(args)
```

Comment:
looks like this already includes the `get_tasks` refactor?

Reply:
Whoops, will merge that in and update this.